archival_README.md

```diff
@@ -0,0 +1,63 @@
+## Overview
```

Comment:
We should be able to just delete this, right? They can always view the contents of README.md in the commit-previous-to-this-change.